tests/benchdnn/inputs/conv/test_conv_ci

```diff
@@ -7,7 +7,7 @@
 --dir=FWD_B,FWD_D
 ### Direct
 --alg=direct
---dt=f32,bf16,f8_e5m2,f8_e4m3,f16,f32:f16:f32,f32:bf16:f32
+--dt=f32,bf16,bf16:bf16:f32,f8_e5m2,f8_e4m3,f16,f32:f16:f32,f32:bf16:f32
```

Comment:
maybe replace `f32:bf16:f32` with `bf16:bf16:f32` in ci coverage ?
@dzarukin for comment

Reply:
`f32:bf16:f32` covers weights up conversion for client systems, should remain.